specs/agents/metrics-otel.md

```diff
@@ -0,0 +1,177 @@
+
+Aggregations are configurable in OpenTelemetry. Currently, this mainly makes sense for Histogram-Instruments. A histogram instrument is usually used when the distribution of the observed values is of interest, e.g. for latency. At the moment the SDK offers [two options](https://opentelemetry.io/docs/reference/specification/metrics/sdk/#histogram-aggregations) for histogram aggregations:
+ * A bucket based histogram with explicitly provided bucket boundaries
+ * An exponential histogram with predefined, exponentially increasing bucket boundaries
+
+In the SDK, the default aggregation to use for each instrument type (counter, gauge, histogram, etc) is defined by the Exporter / MetricsReader. However, users can explicitly override the aggregation per metric by using [Views](https://opentelemetry.io/docs/reference/specification/metrics/sdk/#view). Note that views are an SDK concept and are not part of the OpenTelemetry API.
+
+Agents MUST use the OpenTelemetry SDK [default aggregations per instrument type](https://opentelemetry.io/docs/reference/specification/metrics/sdk/#default-aggregation) as default aggregations for their exporter:
+
+| **Instrument Type**         | **Aggregation**            |
+|-----------------------------|----------------------------|
+|  Counter                    | Sum                        |
+|  Asynchronous Counter       | Sum                        |
+|  UpDownCounter              | Sum                        |
+|  Asynchronous UpDownCounter | Sum                        |
+|  Asynchronous Gauge         | Last Value                 |
+|  Histogram                  | Explicit Bucket Histogram  |
+
+### Histogram Aggregation
+
+The OpenTelemetry SDK uses the following prometheus-inspired bucket boundaries for histograms by default:
+```
+[0, 5, 10, 25, 50, 75, 100, 250, 500, 750, 1000, 2500, 5000, 7500, 10000]
+```
+
+Because IntakeV2 histogram serialization skips empty buckets, we are free to use histogram boundaries which are more accurate and cover a greater range. Therefore, agents SHOULD use the following histogram boundaries by default instead:
+
+```
+[0.00390625, 0.00552427, 0.0078125, 0.0110485, 0.015625, 0.0220971, 0.03125, 0.0441942, 0.0625, 0.0883883, 0.125, 0.176777, 0.25, 0.353553, 0.5, 0.707107, 1, 1.41421, 2, 2.82843, 4, 5.65685, 8, 11.3137, 16, 22.6274, 32, 45.2548, 64, 90.5097, 128, 181.019, 256, 362.039, 512, 724.077, 1024, 1448.15, 2048, 2896.31, 4096, 5792.62, 8192, 11585.2, 16384, 23170.5, 32768, 46341.0, 65536, 92681.9, 131072]
+```
+
+These boundaries are an exponential scale base `sqrt(2)` rounded to six significant figures.
+
+Agents MUST allow users to configure the default histogram boundaries via the `custom_metrics_histogram_boundaries` configuration option:
```

Comment:
nit: https://github.com/elastic/apm/blob/main/specs/agents/configuration.md#adding-a-new-configuration-option suggests a separate section for each defined config var. Perhaps move it to a section below the rest of this `### Histogram Aggregation` section?